oolab/src/main/java/agh/ics/oop/IMapElement.java

```diff
@@ -0,0 +1,6 @@
+package agh.ics.oop;
+
+public interface IMapElement {
+    Vector2d getPosition();
+    String toString();
```

Comment:
`toString()` nie musimy deklarować - ma to każda javowa klasa, odziedziczone z `Object` ;)